.github/actions/setup/action.yml

```
@@ -10,4 +10,12 @@ runs:
       shell: bash
-      run: bun install --frozen-lockfile
\ No newline at end of file
+      run: bun install --frozen-lockfile
+
+    - name: Create environment variables file
+      shell: bash
+      run: |
+        cat << EOF > .env.local
+        OPENAI_API_KEY=sk-proj-1234567890
+        NEXT_PUBLIC_BASE_URL=http://localhost:3000
+        EOF
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)